<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>302</width>
    <height>711</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PARCELAS</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QLineEdit" name="txt_valor">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>121</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(225, 211, 255);</string>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 0, 255);</string>
   </property>
   <property name="text">
    <string>VALOR</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_parcelas">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>60</y>
     <width>121</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(225, 211, 255);</string>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>20</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 0, 255);</string>
   </property>
   <property name="text">
    <string>NÂ° PARCELAS</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_salvar">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>130</y>
     <width>121</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border-radius: 10px;
    background-color: rgb(85, 85, 255);
    color: rgb(255, 255, 255);
}

QPushButton:hover {
    background-color: rgb(100, 100, 255);
}



</string>
   </property>
   <property name="text">
    <string>SALVAR!</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_juros">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>121</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(225, 211, 255);</string>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 0, 255);</string>
   </property>
   <property name="text">
    <string>JUROS %</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="table_parcelas">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>261</width>
     <height>421</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(85, 85, 255);</string>
   </property>
   <column>
    <property name="text">
     <string>VALOR DAS PARCELAS</string>
    </property>
   </column>
  </widget>
  <widget class="QLineEdit" name="txt_total">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>650</y>
     <width>121</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(225, 211, 255);</string>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>650</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 0, 255);</string>
   </property>
   <property name="text">
    <string>TOTAL:</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>txt_valor</tabstop>
  <tabstop>txt_parcelas</tabstop>
  <tabstop>txt_juros</tabstop>
  <tabstop>bt_salvar</tabstop>
  <tabstop>table_parcelas</tabstop>
  <tabstop>txt_total</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
